packages/main/src/util.ts

```diff
@@ -36,3 +37,20 @@ export function findWindow(): Electron.BrowserWindow | undefined {
 }
 
 export const stoppedExtensions = { val: false };
+
+export function getBase64Image(imagePath: string): string | undefined {
+  try {
+    if (fs.existsSync(imagePath)) {
+      const imageContent = fs.readFileSync(imagePath);
+
+      // convert to base64
+      const base64Content = Buffer.from(imageContent).toString('base64');
+
+      // create base64 image content
+      return `data:image/png;base64,${base64Content}`;
+    }
+  } catch (error) {
+    console.error(`Error while creating base64 image content for ${imagePath}`, error);
```

Comment:
wondering if it should send an image displaying 'Not found image' or a default image

Reply:
i think it will should be the caller to handle the response.
In some cases it would be great to show a default icon, in others better not showing anything.

Reply:
anyway it does not prevent to merge PR